doc/api/worker_threads.md

```diff
@@ -1272,6 +1285,14 @@ Calling `unref()` on a worker allows the thread to exit if this is the only
 active handle in the event system. If the worker is already `unref()`ed calling
 `unref()` again has no effect.
 
+### `worker.clearTimer()`
```

Comment:
What is the usecase of exposing this? If it does not exist in childe_process maybe the more complex use cases should be left to be solved with `AbortSignal`?